src/Robo/Inspector/Inspector.php

```diff
@@ -788,7 +788,7 @@ public function issueEnvironmentWarnings($command_name) {
    * @throws \Acquia\Blt\Robo\Exceptions\BltException
    */
   public function warnIfPhpOutdated() {
-    $minimum_php_version = 7;
+    $minimum_php_version = 5.6;
```

Comment:
Why not keep a warning in here, but don't throw an exception? Maybe suggest users to move off 5.6 at their convenience...

Reply:
Or, have a warning and suggest users move _to_ 5.6 if they really need it, but not encourage it as a default. Otherwise, this entire issue feels like a step backwards.

Reply:
This function is not currently a warning but rather an exception, causing BLT to fail. I agree -- giving some warning that you're running 5.6 would be good, although Drupal already does that.